AntistasiOfficial.Altis/CREATE/createSupplyBox.sqf

```diff
@@ -127,24 +126,28 @@ else
 	[[petros,"globalChat","Our watchposts have detected an AAF supply crate. Check your maps!"],"commsMP"] call BIS_fnc_MP;
 };
 
-[false,150,0] params ["_timerRunning","_deploymentTime","_counter"];
-//TODO rework marker
-while {alive _crate AND (_marker in markerSupplyCrates)} do {
+// DEBUG 150 _deploymentTime
+[false,10,0] params ["_timerRunning","_deploymentTime","_counter"];
 
-	// wait until the player loads the crate or have the loaded crate in a city
-	waitUntil {sleep 5;
-		!(isNull attachedTo _crate) OR
-		!({(_crate distance (getmarkerpos _x) < 200)} count ciudades == 0) OR
-		!(_marker in markerSupplyCrates)
+_isCrateUnloaded = false;
+while {alive _crate OR (_marker in markerSupplyCrates)} do {
+
+	// wait until the player loads the crate
+	waitUntil {
+		!(isNull attachedTo _crate)
```

Comment:
marker in markerSupplyCrates condition needed, otherwise it will stay on HQ move and will use CPU and RAM

Reply:
hum you're saying it misses this condition `!(_marker in markerSupplyCrates)` in this waitUntil{} right ?

Reply:
think i see what you mean the script never dies when force deleted (after 45mins of creation) because he is stuck hre https://github.com/A3Antistasi/antistasiofficial/blob/new-develop/AntistasiOfficial.Altis/CREATE/createSupplyBox.sqf#L167.
That's the only step i see all the others have checks on `alive crate` or `!(_marker in markerSupplyCrates)`.